compute_sdk/globus_compute_sdk/serialize/utils.py

```diff
@@ -0,0 +1,32 @@
+def pack_buffers(buffers: list[str]) -> str:
+    """
+    Combines a list of Compute-serialized buffers into a single string format.
+
+    Parameters
+    ----------
+    buffers
+        A list of serialized buffers
+    """
+    packed = ""
+    for buf in buffers:
```

Comment:
return "".join([f"{len(buf)}\n{buf}" for buf in buffers])

That's easier to read for me, but may be too short?  Just personal preference, though.